sdk-setup/src/mb2

```diff
@@ -476,6 +476,16 @@ COMMANDS
         Run the given command on the device specified with '--device'.  Can be
         used for running gdb and a gdb server.
 
+    sign {--set-key | --deploy-key | --import-key |--list-keys }
+        --set-key -n|--name[=<GPG key name>] -p|--passphrase[=<key passphrase>]
+            Set default key for signing rpm packages.
+        --import-key -k|--key[=<GPG key file>] -p|--passphrase[=<key passphrase>]
+            Import local key file to use signing rpm packages.
+        --deploy-key -n|--name[=<GPG key name>]
+            Deploy <name> GPG pubkey to <device>.
+        --list-keys
+            Show all known GPG keys. If device is specified, shows its keys.
+
```

Comment:
Covered before. There is no need for such command(s) on mb2 side. Let me repeat:

`--set-key`

The same is achieved with `alias mb2="mb2 --package-signing-user ... --package-signing-passphrase[-file] ..."` under platform SDK and with `sfdk config --global package.signing-user ...` with Application SDK.

`--import-key`

Platform SDK users may import their keys with `gpg2 --export-secret-keys ID |sdk gpg2 --import` - no need to assist that with yet another command. For Application SDK, when the `package.signing-user <name>` option is set, sfdk should ensure that the corresponding private key is exported as `~/SailfishOS/vmshare/gnupg/<name>.key` prior to executing a command on the build engine, where it should be taken into use by mb2.

`--deploy-key`
`--list-key`

Deploying and listing keys is straightforward already, no need to hide it behind mb2/sfdk:

```
gpg2 --export ID |ssh device sdk-deploy-key -
ssh device sdk-deploy-key --list
```

Similarly with sfdk's assistance for Application SDK users

```
gpg2 --export ID |sfdk device exec sdk-deploy-key -
````

People should be allowed to manage keys on their devices even without the need to use an SDK - and `sdk-deploy-key` already allows that. WRT that it might be also better to name it differently, so that it does not indicate (falsely) an exclusive relation to SDK. I would consider e.g. `rpm-deploy-key`.